docs/OnTrack/Jupyter Notebook/Docker-Documentation-Research-T1-2022.md

```diff
@@ -1,32 +1,67 @@
 # Docker Documentation and Research (WIP) T1-2022
 
-- [Back to Jupyter Notebook Documentation Index](Index.md)
+[Back to Jupyter Notebook Documentation Index](Index.md)
 
-## Conversion Relating to Docker Processes
+## Intro
+
+As we know, the Jupyter Notebook conversion feature will happen via processes that occur inside
+Docker containers. In this document we will discuss the main architecture of this structure in
+relation to OnTrack.
+
+OnTrack is deployed with two main containers that will always be running:
+
+- the frontend container which hosts the logic behind the user interface of OnTrack (mostly
+  unrelated to this feature)
+- the backend container which hosts the logic related to database interactions and other processes
+
+To achieve the Jupyter conversion feature, we will be using a new container that contains all of the
+dependencies (Python, TeX, etc.) needed for a .ipynb to PDF conversion, and performing the
+conversion process within this container. This allows us to have a standalone software package that
+is extendable if required, for example if new Python libraries are required by a unit. It also means
+that we are able to create as many standalone containers as required for different conversion
+processes, such as:
+
+- docx to PDF conversion using Apache POI
+- powerpoint presentation to PDF conversion using Apache POI
+
+This will allow us to add a wide range of compatability in the future, regardless of the software
```

Comment:
Not sure of the use of the word "compatability", maybe "conversion capabilities"? I'm not too sure myself. 
Just a suggestion though